### PR TITLE
Feature/collect category listing

### PR DESCRIPTION
**Pull request by dairbuirabass**
## Description
- Update category tagging
- Implement category export

## Checklist:
- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] I have assigned the correct milestone or created one if non-existent.
- [x] I have correctly labeled this pull request.
- [x] I have linked the corresponding issue in this description.
- [x] I have updated the corresponding Jira ticket.
- [x] I have requested a review from at least 2 reviewers
- [x] I have checked the base branch of this pull request
- [x] I have checked my code for any possible security vulnerabilities

### CODE REVIEW

File: controllers/front/category.php [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+ *
+ * This controller should only be invoked once, when the Nosto module has been installed.
+ */
+class NostoTaggingCategoryModuleFrontController extends NostoTaggingApiModuleFrontController
+{
+    /**
+     * @inheritdoc
+     * @noinspection PhpUnhandledExceptionInspection
+     */
+    public function initContent()
```
Comment (supercid) [inline controllers/front/category.php:41]: Call like http://prestashop.dev.nos.to:8081/en/module/nostotagging/category
You can change the language as well

File: controllers/front/category.php [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    {
+        // We need to forge the employee in order to get a price for a product
+        $employee = new Employee(); //@codingStandardsIgnoreLine
```
Comment (olsi-qose) [inline controllers/front/category.php:44]: Can the category data differ based on a given employee object?

File: controllers/front/category.php [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+ */
+class NostoTaggingCategoryModuleFrontController extends NostoTaggingApiModuleFrontController
+{
+    /**
+     * @inheritdoc
+     * @noinspection PhpUnhandledExceptionInspection
+     */
+    public function initContent()
+    {
+        // We need to forge the employee in order to get a price for a product
```
Comment (olsi-qose) [inline controllers/front/category.php:43]: Update or remove this line

Comment (supercid) [inline controllers/front/category.php:43]:
```suggestion
```

**Review by supercid — APPROVED** [review]

**Pull request merged**